extension/storage/dbstorage/README.md

```diff
@@ -14,23 +14,25 @@
 
 > :construction: This extension is in alpha. Configuration and functionality are subject to change.
 
-The Database Storage extension can persist state to a relational database. 
+The Database Storage extension can persist state to a relational database.
 
 The extension requires read and write access to a database table.
 
-`driver`: the name of the database driver to use. By default, the storage client supports "sqlite3" and "pgx".
+`driver`: the name of the database driver to use. By default, the storage client supports "sqlite" and "pgx".
 
 Implementors can add additional driver support by importing SQL drivers into the program.
 See [Golang database/sql package documentation](https://pkg.go.dev/database/sql) for more information.
 
 `datasource`: the url of the database, in the format accepted by the driver.
 
+ >**NOTE:** If you are using legacy driver `sqlite3` and have additional driver parameters in `datasource` - please follow [migration guide](#migration-guide-from-sqlite3-to-sqlite-driver-options) to update your configuration.
+`db_storage` component has compatibility convertor in place, but it's better to review and update used driver parameters.
```

Comment:
should the changelog indicate the change as breaking?

Reply:
I'm still not confident that this change is breaking, because compatibility layer allows graceful migration from one driver to another. But if breaking change label will bring more user attention to this change - it sounds reasonable to make it breaking

Reply:
Updated changelog type to breaking